monitor/visualizer.py

```diff
@@ -0,0 +1,163 @@
+import json
+import os
+import requests
+
+from monitor.data_loader import LOG_PREFIX
+from common.logger import Logger
+
+logger = Logger(actor="Visualizer", log_name_prefix=LOG_PREFIX)
+
+
+class Visualizer:
+    def __init__(self, kibana_host, kibana_port, max_users):
+        self.kibana_endpoint = "http://{}:{}".format(kibana_host, kibana_port)
+        self.saved_obj_endpoint = "{}/api/saved_objects/".format(self.kibana_endpoint)
+        self.space_endpoint = "{}/api/spaces/space/".format(self.kibana_endpoint)
+        self.headers = headers = { 
+            "Content-Type": "application/json",
+            "kbn-xsrf": "true"
+        }
+        self.max_users = max_users
+
+    def create_dashboard(self):
+        index_pattern = None
```

Comment:
This initialisations do not seem necessary. A dict returns `None` if a value is not found when calling get.